### DISCUSSION TITLE
[Suggestion] .vsmeta importer

### DISCUSSION THREAD

**Discussion by Speiger** [jellyfin/jellyfin#12710] [General]
o/ I know the issue says you shouldn't create suggestion issues but i think this is a exception that can be made.

I am working on a vsmeta importer/exporter, that is already able to parse said files, for my custom tool i planned to use for bulk creation of said files.
I have seen that someone [asked](https://features.jellyfin.org/posts/2853/support-vsmeta-files) for adding vsmeta support.

And I thought ok, since i am already creating an importer for myself (written in java) why not share it if there is interest for it.

I sadly don't have the time to contribute to this project directly via creating a PR, but would there be interest if I just either link the repo here or dump the file that just parses the file here and someone implements it.
Under the assumption that it would be integrated at some point?

I am asking, mainly because atm i am still using VideoStation and i have my reasons for it, but i am considering assuming Synology isn't walking back in the next few months to switch to jellyfin and see if it is good to switch to.

TL:DR I have a working VsMeta Parser and wanted to know, if you want to have the source so you can integrate it, since i saw interest for an automated conversion from Synology VideoStation to Jellyfin.

**Comment by gnattu** [comment]
This should be implemented as a metadata source plugin and not integrated into the server itself in my opinion. Our ABI for 10.10 is now basically locked so if someone is interested in making a plugin it is a good time to start now.

**Reply by Speiger** [reply]
@gnattu well I sadly don't have time for making a plugin. But if someone is interested in making one and wants the source he can ping me here so we can get that started ^^"

Anyways thanks for the answer.
